bigml/model.py

```diff
@@ -373,11 +373,16 @@ def predict_probability(self, input_data, by_name=True,
             else:
                 output = {'prediction': prediction}
         else:
-            instances = 1.0
             root_dist = self.tree.distribution
-            total = float(sum([category[1] for category in root_dist]))
-            category_map = {category[0]: category[1] / total
+
+            if not self.tree.weighted:
```

Comment:
you can ignore this nit, but I would reverse the condition to avoid an additional not

```python
if self.tree.weighted:
    ...
else:
  ...
```